src/request/request-tab.js

```diff
@@ -43,3 +44,10 @@ module.exports.registerTab({
     title: 'Request',
     component: request.Request
 });
+
+module.exports.registerTab({
+    key: 'tab.logging',
+    title: 'Trace',
+    component: LoggingComponent,
+    viewModel: LoggingViewModel
```

Comment:
I wonder, to make the registrations a little cleaner, if the requires should just be used in place or if we want to put them into a var, just above the matching `registerTab`.